WebRoot/img/zimbra/ImgEditorDrag.svg

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generated by IcoMoon.io -->
```

Comment:
Can you clean up this svg code to have just `svg` tag and not `doctype` and `xml` attribute.
Refer other for ref. 

Also please see: https://github.com/Zimbra/zm-web-client/pull/6/files#diff-c222f38db9974ef443747ae87b6c374cR1 as reference to remove `xmlns:xlink="http://www.w3.org/1999/xlink" width="32" height="32" `